### ISSUE TITLE
Friends Tab Rework Submission #1 - Guest, Owner & Moderator Views + Scenarios

### ISSUE THREAD

**Issue by samgawaran** [colonistio/design#122]
## Summary
Added moderator view and fixed some design inconsistencies in the previous submissions

### Logic/Terms

`Friends` - You and another player both follow each other (only tab available in guest view)
`Following` - Players you follow
`Fans` - Players following you
`Blocked Players` - Players you blocked
`Blockers` - Players who blocked you (available for moderators only)

## Details
https://www.loom.com/share/5ae97067f7b24faca0058f79a5b4bcd9
____________________________________________________

### Guest View
![image](https://user-images.githubusercontent.com/90673012/182177888-09f94b9b-40e7-491b-ac0e-4b23a1d14c11.png)

### Owner View
![image](https://user-images.githubusercontent.com/90673012/182177988-51dbc258-385f-4ae5-bf8e-5910e71ce5f9.png)

### Moderator View
![image](https://user-images.githubusercontent.com/90673012/182178060-4cf96511-19e0-430a-8d20-59cc1a8f8cf5.png)

### Fans & Followers into Friends
![Fan to Friends](https://user-images.githubusercontent.com/90673012/182182107-0f7e40e0-652d-4230-820b-b1d4d6bec358.png)

Following - If a player that you're following follows you back, they become your friends
Fans - Following a Fan makes them your friend

### Blocking a Player
![image](https://user-images.githubusercontent.com/90673012/182182661-575e7a2c-b920-4ae4-b368-a51cb7b096a6.png)

### Reporting a Player
![image](https://user-images.githubusercontent.com/90673012/182187162-88b9dff1-46b9-4856-b90f-a6e1f2aa3509.png)

### Unblocking a Player
![image](https://user-images.githubusercontent.com/90673012/182183822-4ad1d043-e8db-4fc4-86e7-2039becf61cc.png)

### Unfollowing a Player
![image](https://user-images.githubusercontent.com/90673012/182184194-97c6760c-98de-40d8-961f-70e0b7e95fbd.png)

- Players can "Follow" the player back immediately after they another player if they don't go to another page or click on refresh

## Related Submissions
- https://github.com/colonistio/design/issues/121
- https://github.com/colonistio/design/issues/112

## Related links

- https://github.com/colonistio/design/issues/2

## Resource
- Figma Link: https://www.figma.com/file/20FbrQgSFzrbNvBl7esLNf/Colonist-%2F-Profile-Tab?node-id=105%3A33001

## Submission Checklist

- [x] This submission abides by [Des-lvl-1](https://github.com/colonistio/katan/blob/development/docs/process/designer-level-1.md), [Des-lvl-2](https://github.com/colonistio/katan/blob/development/docs/process/designer-level-2.md) & [Des-lvl-3](https://github.com/colonistio/katan/blob/development/docs/process/designer-level-3.md)

**Comment by samgawaran** [comment]
@goktugyil @demiculus @JuanRoman77 

review this issue first before I add the specifics here: https://github.com/colonistio/katan/issues/7822

**Comment by demiculus** [comment]
1. Fans -> Followers
2. I don't know what this button does: https://i.imgur.com/yo3KzAX.png it should write unfollow 
3. Make all button green, I get more confused whenI see orange & green used in random ways
4. Should write follow back https://i.imgur.com/Us5OYRB.png

**Comment by JuanRoman77** [comment]
> Friends - You and another player both follow each other (only tab available in guest view)
> Following - Players you follow
> Fans - Players following you

Disagree. Better to only have 1 category: Friends
So you can add to your friends' list anyone you want, without the need of a notifications system, accepting/denying, following back, etc. Just and Add/Remove Friend button in the /prifle.

This reduces the complexity by a lot and probably better UX.

@goktugyil still waiting for your approval about this.

**Comment by samgawaran** [comment]
Added the fixes in this new issue: https://github.com/colonistio/design/issues/123 also added Juan's idea there.

**Comment by goktugyil** [comment]
> So you can add to your friends' list anyone you want, without the need of a notifications system, accepting/denying, following back, etc. Just and Add/Remove Friend button in the /prifle.

How is that getting rid of notification system? 100 people randomly added me to their friends list? I have to review them and accept them. I don't want people to customly add them selves into my friends list

**Comment by JuanRoman77** [comment]
> > So you can add to your friends' list anyone you want, without the need of a notifications system, accepting/denying, following back, etc. Just and Add/Remove Friend button in the /prifle.
> 
> How is that getting rid of notification system? 100 people randomly added me to their friends list? I have to review them and accept them. I don't want people to customly add them selves into my friends list

They won't add themselves to your list!!! They will add you to theirs. And you can add whoever you want as well, without them knowing. That's what I suggested.
